_ingest-pipelines/processors/sort.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: default
+title: Sort
+parent: Ingest processors
+nav_order: 250
+---
+
+# Sort processor
+
+The `sort` processor sorts an array of items in either ascending or descending order. The processor takes in a field name (`field`) and a sort order (`order`), and it creates a new field (`target_field`) with the sorted array.
```

Comment:
```suggestion
The `sort` processor sorts an array of items in either ascending or descending order. Numeric arrays are sorted numerically, while string or mixed arrays (strings and numbers) are sorted lexicographically. Throws an error if the input is not an array.
```

Reply:
@gaobinlong Please confirm this change addresses your comments accurately. Thank you.